mackerel-plugin-snmp/lib/snmp.go

```diff
@@ -52,8 +52,11 @@ func (m SNMPPlugin) FetchMetrics() (map[string]interface{}, error) {
 
 		ret, err := strconv.ParseFloat(fmt.Sprint(resp.Variables[0].Value), 64)
 		if err != nil {
-			log.Println(err)
-			continue
+			ret, err = strconv.ParseFloat(fmt.Sprintf("%s", resp.Variables[0].Value), 64)
+			if err != nil {
+				log.Println(err)
+				continue
+			}
```

Comment:
IMO, It's hard to understand immediately why second sentence is needed.

Can you add a comment, or add an explain text in commit message?

Reply:
Sure, I added.